packages/cloudflare/src/entrypoints/image-service.ts

```diff
@@ -0,0 +1,40 @@
+import type { ExternalImageService } from 'astro';
+
+import { baseService } from 'astro/assets';
+import { isESMImportedImage, isRemoteAllowed, joinPaths } from '../utils/assets.js';
+
+const service: ExternalImageService = {
+	validateOptions: baseService.validateOptions,
```

Comment:
You might want to spread `baseService` instead of putting the options one by one